pyfar/classes/coordinates.py

```diff
@@ -1374,16 +1375,313 @@ def show(self, mask=None, **kwargs):
+                    radius. Differences are larger than 1e-15")
+            radius = np.max(radius)
+
+            if distance_measure == 'angular':
+                # convert angle in radiant to distance on the sphere
+                # d = 2r*pi*d/(2*pi) = r*d
+                distance = radius * distance
+
+            # convert cartesian coordinates to length on the great circle
+            distance = 2 * radius * np.arcsin(distance/(2*radius))
+
+        if self.cdim == 1:
+            if k > 1:
+                index_multi = np.moveaxis(index, -1, 0)
+                index = np.empty((k), dtype=tuple)
+                for kk in range(k):
+                    index[kk] = tuple([index_multi[kk]], )
+            else:
+                index = tuple([index], )
+        else:
+            index_array = np.arange(self.csize).reshape(self.cshape)
+            index_multi = []
+            for dim in range(self.cdim):
+                index_multi.append([])
+                for i in index.flatten():
+                    index_multi[dim].append(np.where(i == index_array)[dim][0])
+                index_multi[dim] = np.asarray(
+                    index_multi[dim]).reshape(index.shape)
+            if k > 1:
+                index_multi = np.moveaxis(index_multi, -1, 0)
+                index = np.empty((k), dtype=tuple)
+                for kk in range(k):
+                    index[kk] = tuple(index_multi[kk])
+            else:
+                index = tuple(index_multi)
+
+        if k > 1:
+            distance = np.moveaxis(distance, -1, 0)
+
+        return index, distance
+
+    def find_within(
+            self, find, distance=0., distance_measure='euclidean',
+            atol=1e-15, return_sorted=True):
+        """
+        Find coordinates within a certain distance to the query points.
+
+        Parameters
+        ----------
+        find : pf.Coordinates
+            Coordinates to which the nearest neighbors are searched.
+        distance : number, optional
+            Maximum allowed distance to the given points ``find``.
+            Distance must be >= 0. For just exact matches use ``0``.
+            The default is ``0``.
+        distance_measure : string, optional
+            ``'euclidean'``
+                distance is determined by the euclidean distance.
+                This is default.
+            ``'spherical'``
+                distance is determined by the great-circle distance.
+            ``'angular'``
+                distance is determined by the angles in radiant.
+        return_sorted : bool, optional
+            Sorts returned indices if True and does not sort them if False.
+            The default is True.
+
+        Returns
+        -------
+        index : tuple of array
+            Indices of the containing coordinates. Arrays of shape
+            (find.cshape).
+
+        Notes
+        -----
+        This is a wrapper for ``scipy.spatial.cKDTree``. Other than previous
```

Comment:
```suggestion
        This is a wrapper for ``scipy.spatial.cKDTree``. Compared to previous
```
Minor rewording suggestion.

Reply:
Or did you change this with Fabians comment?